test/cli/package-autocorrect-missing-import/test.out

```diff
@@ -1,57 +1,52 @@
-foo_class.rb:7: No import provides `Foo::Bar` https://srb.help/5002
+foo_class.rb:15: Unable to resolve constant `MyClass` https://srb.help/5002
+    15 |  Foo::Bar::MyClass::SUBCLASSES # resolves via root
+          ^^^^^^^^^^^^^^^^^
+  Autocorrect: Done
+    foo_class.rb:15: Replaced with `Class`
+    15 |  Foo::Bar::MyClass::SUBCLASSES # resolves via root
+          ^^^^^^^^^^^^^^^^^
+    https://github.com/sorbet/sorbet/tree/master/rbi/core/class.rbi#L71: `Class` defined here
+    71 |class Class < Module
+        ^^^^^^^^^^^^^^^^^^^^
+  Autocorrect: Done
+    foo_class.rb:15: Replaced with `Digest::Class`
+    15 |  Foo::Bar::MyClass::SUBCLASSES # resolves via root
+          ^^^^^^^^^^^^^^^^^
+    https://github.com/sorbet/sorbet/tree/master/rbi/stdlib/digest.rbi#L176: `Digest::Class` defined here
+     176 |class Digest::Class
+          ^^^^^^^^^^^^^^^^^^^
+
+foo_class.rb:7: No import provides `Foo::Bar::OtherPackage::OtherClass` https://srb.help/3718
      7 |      Foo::Bar::OtherPackage::OtherClass # resolves via root
-              ^^^^^^^^
+              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Done
     __package.rb:6: Inserted `import Foo::Bar::OtherPackage`
      6 |  # import Foo::Bar::OtherPackage ## MISSING!
                                                      ^
-    other/__package.rb:3: Defined here
-     3 |class Foo::Bar::OtherPackage < PackageSpec
-        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'm not sure if i mentioned this on another comment, but could we preserve this error section too?

Reply:
Done!